assets/components/ovn/master/daemonset.yaml

```diff
@@ -169,23 +169,23 @@ spec:
               - |
                 set -x
                 rm -f /var/run/ovn/ovnnb_db.pid
-                  echo "$(date -Iseconds) - nbdb - postStart - waiting for master to be selected"
-
-                  # Upgrade the db if required.
-                  DB_SCHEMA="/usr/share/ovn/ovn-nb.ovsschema"
-                  DB_SERVER="unix:/var/run/ovn/ovnnb_db.sock"
-                  schema_name=$(ovsdb-tool schema-name $DB_SCHEMA)
-                  db_version=$(ovsdb-client -t 10 get-schema-version "$DB_SERVER" "$schema_name")
-                  target_version=$(ovsdb-tool schema-version "$DB_SCHEMA")
-
-                  if ovsdb-tool compare-versions "$db_version" == "$target_version"; then
-                    :
-                  elif ovsdb-tool compare-versions "$db_version" ">" "$target_version"; then
-                      echo "Database $schema_name has newer schema version ($db_version) than our local schema ($target_version), possibly an upgrade is partially complete?"
-                  else
-                      echo "Upgrading database $schema_name from schema version $db_version to $target_version"
-                      ovsdb-client -t 30 convert "$DB_SERVER" "$DB_SCHEMA"
-                  fi
+                echo "$(date -Iseconds) - nbdb - postStart - waiting for master to be selected"
```

Comment:
A shell script this complex would be easier to review as a separate file where we would get syntax highlighting and not have the indentation introduced by the YAML. Changing that shouldn't hold up this PR, since it's obviously like that already, but maybe we can move it later on?

Reply:
Yes, it is the same as how ovnk manifests are maintained in openshift (cluster-network-operator). I created the sub-task for later change: https://issues.redhat.com/browse/USHIFT-235